lib/semian/atomic_enum.rb

```diff
@@ -0,0 +1,37 @@
+module Semian
+  class AtomicEnum < AtomicInteger #:nodoc:
+    undef :increment_by
+    undef :increment
```

Comment:
Yeah. On the other hand, it wouldn't actually be used anywhere. I'm somewhat indifferent.